master/internal/command/command_job_service.go

```diff
@@ -0,0 +1,67 @@
+package command
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	"github.com/determined-ai/determined/master/internal/config"
+	"github.com/determined-ai/determined/master/internal/rm/rmerrors"
+	"github.com/determined-ai/determined/master/internal/sproto"
+	"github.com/determined-ai/determined/proto/pkg/jobv1"
+)
+
+// job.Service methods
+
+// ToV1Job() takes a command and returns a job.
+func (c *command) ToV1Job() *jobv1.Job {
```

Comment:
I think we want to lock over these public functions

Reply:
would these locks come from the job service manager or from the command module?

Reply:
from command (or experiment for that code). basically, we want to provide the same serialized data operation & isolation that channels and the actor system provide.

Reply:
makes sense, just like all the other actor refactors